src/Extensions/SiteTreeContentReview.php

```diff
@@ -397,7 +397,7 @@ public function updateSettingsFields(FieldList $fields)
             $options
         );
 
-        $users = Permission::get_members_by_permission(["CMS_ACCESS_CMSMain", "ADMIN"]);
+        $users = Permission::get_members_by_permission(['CMS_ACCESS_CMSMain', 'CMS_ACCESS_LeftAndMain', 'ADMIN']);
```

Comment:
Why is this being added? This doesn't look like something that should be covered by unit test and shouldn't be in a MNT commit.

Reply:
I'll move this to a separate bugfix pr